slack_sdk/web/async_client.py

```diff
@@ -1601,6 +1601,83 @@ async def auth_test(
         """
         return await self.api_call("auth.test", params=kwargs)
 
+    async def bookmarks_add(
+        self,
+        *,
+        channel_id: str,
+        title: str,
+        type: str,
+        emoji: Optional[str] = None,
+        entity_id: Optional[str] = None,
+        link: Optional[str] = None,  # include when type is 'link'
```

Comment:
This comment will be removed when the codegen command is executed next time. Can you add this change to `client.py` and run either `python setup.py codegen` or `./scripts/run_validation.sh`?

Reply:
Will do! I am also working on an integration test (mostly so I can have some more hands-on experience with what this API does), so I will include this suggested change as well as the integration test in a moment, and ask for your review again 🙇

Reply:
@filmaj Got it. Thanks!

Reply:
Good call out @seratch I had meant to leave this in the the client.py file!

Reply:
FYI when I moved the comment to `client.py` and re-ran `python3 setup.py codegen`, the comment only shows up in `legacy_client.py` and not `async_client.py` 🤔